cluster/gce/util.sh

```diff
@@ -1116,6 +1116,7 @@ NODE_PROBLEM_DETECTOR_VERSION: $(yaml-quote ${NODE_PROBLEM_DETECTOR_VERSION:-})
 NODE_PROBLEM_DETECTOR_TAR_HASH: $(yaml-quote ${NODE_PROBLEM_DETECTOR_TAR_HASH:-})
 NODE_PROBLEM_DETECTOR_RELEASE_PATH: $(yaml-quote ${NODE_PROBLEM_DETECTOR_RELEASE_PATH:-})
 NODE_PROBLEM_DETECTOR_CUSTOM_FLAGS: $(yaml-quote ${NODE_PROBLEM_DETECTOR_CUSTOM_FLAGS:-})
+CNI_STORAGE_PATH: $(yaml-quote ${CNI_STORAGE_PATH:-kubernetes-release/network-plugins})
```

Comment:
Should we follow the pattern used by the other envs?
```sh
CNI_STORAGE_PATH: $(yaml-quote ${CNI_STORAGE_PATH:-})
```
Since the default has been given by `config-default.sh` and `config-test.sh`?

Reply:
good idea, thanks.